docs/src/operations/best-practices/monitoring.md

```diff
@@ -45,7 +45,7 @@ To send validator health notifications to your Telegram account, we are going to
 
 #### Create a Bot Using BotFather
 
-In Telegram, search for `@BotFather`. Send the following message to _@BotFather_: `/newbot`.
+In Telegram, search for `@BotFather` or open [`t.me/botfather`](t.me/botfather). Send the following message to _@BotFather_: `/newbot`.
```

Comment:
Can you revert this line and leave the PR focused on `Solana watchtower` ==> `Agave watchtower` changes ?